ci/deps/actions-311.yaml

```diff
@@ -43,7 +43,7 @@ dependencies:
   - pyarrow>=7.0.0
   - pymysql>=1.0.2
   - pyreadstat>=1.1.5
-  # - pytables>=3.7.0, 3.8.0 is first version that supports 3.11
+  - pytables>=3.8.0
```

Comment:
We need to wait until we bump the minimum version of pytables in order to change this

Reply:
No, we probably have to do 2 things here:

- Pin this to greater or equal than 3.8 since pytables 3.7 does not support python 3.11 (which is something we can't do anything about)
- Fix the sync min version script to allow this

Reply:
I am a bit vary about bumping the min version.

Is there an easy way to add an ignore option to the script? (Haven't looked at it, so no clue)

Reply:
> Is there an easy way to add an ignore option to the script? (Haven't looked at it, so no clue)

I don't think so as different minimum versions per Python version was not anticipated when making the script. I would prefer the consistency over starting to add exceptions (and in this case I'm sure it's safe to assume if it works in older Python versions, it should work in Python 3.11)

Reply:
Not totally sure if I understand you correctly, but pytables is segfaulting with 3.7 for Python 3.11

Reply:
I meant that e.g. pandas w/ pytables 3.8 + python 3.11 should behave similarly as pandas w/ pytables 3.7 + python 3.9 for example, so I think the pandas w/ pytables is sufficiently "tested" without having these tests run on 3.11

Reply:
Ah ok, I'd be ok with this